src/main/java/uk/gov/ons/ctp/response/casesvc/domain/repository/CaseRepository.java

```diff
@@ -61,7 +61,7 @@
    * @param iac the iac
    * @return the cases associated with the IAC (see above)
    */
-  List<Case> findByIac(String iac);
+  List<Case> findByIacAudits_iac(String iac);
```

Comment:
Not sure this name change of the method helps the clarity of what it's doing

Reply:
LOL it's one of those JPA query methods where the it works out what to get from the method name ... it has to be called that